RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+More internal refactoring that should have no user-visible effect.
```

Comment:
Worth mentioning something similar to in the PR?

Reply:
Ah yeah, will do next time.